### DISCUSSION TITLE
Ability to hide sources of subscribed feeds in my main feed (All feeds)

### DISCUSSION THREAD

**Discussion by Terkyz** [spacecowboy/Feeder#122] [Ideas]
Many thanks to @spacecowboy for opening the Feeder repository discussions, as I have long wanted to request this feature but didn't have how.

My idea is like this: Suppose I subscribe to a feed source (RSS) but I don't want to have it always present, only on some  occasions  for one reason or another, so my request is the ability to hide that feed from my main feed also called "All feeds".

To give a little more context, I will give a personal example. I generally follow technology feeds but I want to be subscribed to news feeds to keep up with what's going on, but most news feeds FLOOD me with feeds, if there was a way to hide the feed of a certain source I could hide the feed of those news pages and when I want to read those feeds I would go directly to the feed with no problem.

**Comment by Mr-Bajs** [comment]
How differs this from the already categories that you can tag your feeds when you add them? Have you tried that?
I have several catagories for my rss feed and it works exellent to keep my feed in order.

**Comment by Terkyz** [comment]
Yes, actually I have 6 different categories, the problem is that personally I don't visit tag by tag, but I see all my feeds thru the "All feeds" section.

Although now that you say it, I suppose I could create an exclusive tag for the feeds I want to read, and read exclusively from that category, although it doesn't exactly solve my problem, thanks for the idea.

**Comment by spacecowboy** [comment]
Personally I keep three primary tags:

* `Don't miss` - these are low frequency feeds where I really want to read/skim every single entry
* `Daily` - feeds which spam a bit more, where I will browse and maybe read some of it
* `Bored` - really spammy feeds, which I only read if I ran out of other things to read

**Reply by spacecowboy** [reply]
And I never open the All Feeds tags

**Reply by Terkyz** [reply]
Is there a way to define more than one tag to the same feed? I am trying to add my feeds to a "General" category without messing up my categories, but when I try to do it using a comma (For example: Linux, PC) everything is part of the same tag being now "Linux, PC" and not "Linux" one tag and "PC" another.

**Reply by spacecowboy** [reply]
No, it's one tag per feed I'm afraid.

**Reply by flatline-studios** [reply]
This is the biggest issue with this... If you can only have one tag, then it's not a `tag`, it's a `label`.

We need true tagging abilities and/or tags should be renamed to `label`s.

**Reply by spacecowboy** [reply]
If you need it, then go ahead and build it

**Comment by c-andy-candies** [comment]
Hey just wanted to let you know I created PR #185 that adds a setting for this
